app/features/atp/passport/ui/idx/controller.ts

```diff
@@ -193,12 +193,21 @@ const postPassport = async (
     identityEvidence.atpResponse = decoded;
     req.session.identityEvidence = req.session.identityEvidence || [];
```

Comment:
Also, there's another bug we all missed when writing this previously - this should be put onto the `req.session.sessionData`, not directly onto the `req.session`. It doesn't matter too much in this site, but `core-front` is looking for the data in a different place than where we put it :)